### DISCUSSION TITLE
It's possible to update password via Github API?

### DISCUSSION THREAD

**Discussion by adryells** [community/community#132521] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I'm starting a new project and I would like to know if it is possible to change my own password using the Github API. 
If yes, how? I didn't find any docs about it.

**Comment by harshjuly12** [comment, marked as answer]
Hey @adryells if im correct changing your pwd via using the GitHub API is not directly supported, you will have to do it manually through the GitHub website

**Reply by adryells** [reply]
Okay, thanks @harshjuly12 !
